docs/manuals/core/config/scenario-config.md

```diff
@@ -64,3 +64,16 @@ associated to a Cycle corresponding to their creation date. See documentation on
 [Scenario and cycle management](../entities/scenario-cycle-mgt.md).
 
 [:material-arrow-right: The next section introduces the global configuration](global-config.md).
+
+# Configuring scenario comparators
+
+Let us imagine a common situation where the pipeline configuration `pipeline_config` has been created with `datanode_config` as one of the data node configuration for this the pipeline configuration.
+The function `compare_function` has also been defined:
```

Comment:
I believe we should add more details on `compare_function`. Indeed, this `compare_function` must be able to accept as a parameter a list of data to compare. Each data from this list is an output of the data node read method, where the data node is instantiated from `datanode_config`.

Maybe we can take an example.